testsrc/README.md

```diff
@@ -32,14 +35,74 @@ This behavior can be changed through different means:
 
 ## Running a specific TestSuite
 ```
-./gradlew test --tests org.mozilla.javascript.tests.Test262SuiteTest 
+./gradlew test --tests org.mozilla.javascript.tests.Test262SuiteTest
+```
+
+## test262.properties format
+The [test262.properties](test262.properties) file is used to specify which tests from the official ECMAScript Test Suite to include/exclude,
+so that the test suite can pass even though Rhino is not yet 100% standards compliant
+
+The [test262.properties](test262.properties) file:
+- lists the subfolders of the [test262](../test262) folder to include or exclude when running tests
+- lists the .js test files that are expected to fail per (sub)folder
+
+**Basics**
+```
+built-ins/Array <- include all the tests in the test262/built-ins/Array folder
+    from/calling-from-valid-1-noStrict.js <- but expect that this tests fails
+```
+If `built-ins/Array/from/calling-from-valid-1-noStrict.js` indeed fails, it wont fail the test suite. If it passes, this will be logged while running the test suite
+
+**Skipping entire folders**
+```
+~built-ins/decodeURI
+    name.js
+    S15.1.3.1_A2.4_T1.js
+    S15.1.3.1_A5.2.js
+```
+Folders can be prefixed with a `~` to mark the folder to be skipped entirely, for example because it contains all tests for a feature not yet supported by Rhino or because running the tests take a long time.
```

Comment:
If the tilde marks the whole folder to be skipped, doesn't that imply that all the files in it would also be skipped?

In other words, in the example above, wouldn't the same behavior result if the three individual file names (name.js and so on) were not in the file?

Reply:
yes, but `built-ins/decodeURI` is excluded because of performance reasons. The three files were failing at the time when the entire folder was excluded (or so I assumed). And I decided we do not want to loose this info (as it was there in the file before I made changes

Reply:
Does that mean on regeneration if the directory is marked as skipped, then it retains the files listed below it?

Reply:
Yup: marking a folder to be skipped using the ~ is a manual action. If you as a dev decide to not remove the failing files of that folder, then when regenerating the file, it'll copy them over

As I mentioned in the description of the PR and the README: the only things list when regenerating the file are lines that just contain comments (first non-whitespace character on the line being either an ! or a #)